modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -253,6 +253,14 @@ public static Properties readProperties(Path path) throws IOException {
 		return properties;
 	}
 
+	public static String separatorsToUnix(String path) {
+		if ((path != null) && (path.indexOf(92) != -1)) {
```

Comment:
Can we remove the `path.indexOf(92) != -1` check and simplify the code like this:

```
if ((dirPathString != null) && (File.separatorChar == '\\')) {
	dirPathString = dirPathString.replace('\\', '/');
}
```